src/Header.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+
+let Header = () => 
+    <div className="header">
+        <Link to='/'>Home</Link>
+        <Link to='/category/'>Category</Link>
```

Comment:
I would change this to "set" instead of "category" to keep vocab consistent.

Reply:
Made change!